### PR TITLE
Added environments for encrypt-decrypt

### PR DESCRIPTION
**Pull request by benlstanley**

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -48,7 +48,63 @@ steps:
...
+    env:
+      REACT_APP_VIRTRU_ENV: 'production'
+    agents:
+      - *agent_queue
+
+  - label: ':npm: NPM Build (Staging)'
+    command: 'buildkite-scripts/scripts/npm-build.sh'
+    branches: 'develop'
+    env:
+      REACT_APP_VIRTRU_ENV: 'staging'
```
Comment (jrschumacher) [inline .buildkite/pipeline.yml]: @benlstanley this env is not needed

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -48,7 +48,63 @@ steps:
...
+    agents:
+      - *agent_queue
+
+  - label: ':npm: NPM Build (Develop)'
+    command: 'buildkite-scripts/scripts/npm-build.sh'
+    branches: '!master !develop'
+    env:
+      REACT_APP_VIRTRU_ENV: 'develop'
```
Comment (jrschumacher) [inline .buildkite/pipeline.yml]: Remove

File: .buildkite/pipeline.yml [hunk 3]
```diff
@@ -48,7 +48,63 @@ steps:
       - *agent_queue
 
   - wait
+  - label: ':npm: NPM Build (Production)'
+    command: 'buildkite-scripts/scripts/npm-build.sh'
+    branches: 'master'
+    env:
+      REACT_APP_VIRTRU_ENV: 'production'
```
Comment (jrschumacher) [inline .buildkite/pipeline.yml]: Remove

File: .buildkite/pipeline.yml [hunk 4]
```diff
@@ -48,7 +48,63 @@ steps:
...
+    agents:
+      - *agent_queue
+
+  - block: 'Unblock Deploy'
+
+  - block: 'Check your branch name'
+    prompt: 'Your branch name does not match expected options. This will not result in a successful deployment unless you have explicitly defined environment variables.'
+    branches: '!master !develop !*release/* !*feature/* !*hotfix/* !*bug/*'
 
+  # Build and Deploy (Develop)
```
Comment (jrschumacher) [inline .buildkite/pipeline.yml]: Need to add a invalidate cache

File: .buildkite/pipeline.yml (hunk 3 above)
Comment (benlstanley) [reply .buildkite/pipeline.yml]: so instead of REACT_APP_VIRTRU_ENV it should be NODE_ENV? for all fo them?

File: .buildkite/pipeline.yml [hunk 5]
```diff
@@ -48,7 +48,63 @@ steps:
...
+  - label: ':s3: Deploy to S3 Develop'
+    command: 'buildkite-scripts/scripts/developer-demos/deploy-to-s3.sh'
+    branches: '!master !develop'
+    env:
+      S3_PATH: 'dd-develop'
+    agents:
+      - *agent_queue
   - label: ":s3: Invalidate CloudFront Cache"
```
Comment (jrschumacher) [inline .buildkite/pipeline.yml]: Go ahead and add a production one too

File: .buildkite/pipeline.yml [hunk 6]
```diff
@@ -48,7 +48,63 @@ steps:
       - *agent_queue
```
Comment (jrschumacher) [inline .buildkite/pipeline.yml]: Remove line 43 - 48

**Comment by jrschumacher** [comment]
@NathanAB would you review?

**Comment by jrschumacher** [comment]
@benlstanley I've update the pipeline cause there was minor syntax issues.

**Review by NathanAB — APPROVED** [review]

**Review by jrschumacher — APPROVED** [review]

**Pull request merged**